File: Nível 1/1113 - Crescente e Decrescente/main.lua
while true do
    valor = io.read()
    x1, x2 = valor:match("(%S+)%s+(%S+)")
    x1 = tonumber(x1)
    x2 = tonumber(x2)

    if x1 == x2 then 
        break
    else
        if x1 > x2 then 
            print("Decrescente")
        elseif x2 > x1 then
            print("Crescente")
        end
    end
end

File: Nível 1/1113 - Crescente e Decrescente/main.py
while True:
    valor = input("A: ")
    x1,x2 = valor.split(" ")
    x1 = int(x1)
    x2 = int(x2)

    if x1 == x2:
        break 
    else: 
        if x1 > x2:
            print("Decrescente")
        elif x2 > x1:
            print("Crescente")